### ISSUE TITLE
Disable the "Follow Fedora branching" feature while creating Copr projects

### ISSUE THREAD

**Issue by praiskup** [packit/packit-service#1651]
For non-Packit use cases, it's best to have this feature enabled.  Users
that enable Rawhide chroots typically want us (Copr) to automatically
branch the Rawhide builds to a "branched" chroot, once it appears.

But Packit handles the set of chroots itself when the build is done.
There's probably no motivation to let us copy the builds.  At the time of
branching, disabling the feature would save us a lot of copying work and
we would save some storage (tens of hundreds of GB).

**Comment by lachmanfrantisek** [comment]
Yes, we can do that. (If Copr provides an API for that...;) Even this is important mainly to the non-temporary projects which is probably a minority of all the projects...

We can also decrease the number of days the temporary project is removed:
https://github.com/packit/packit/blob/41a74aefaa6a7d07f2559513c0d454d082b1efdd/packit/copr_helper.py#L121-L123

**Comment by praiskup** [comment]
If I'm not wrong we are missing that API, yes.  Reported here https://pagure.io/copr/copr/issue/2312

**Comment by mfocko** [comment]
waiting for the Copr deployment on 5/25 to be able to get the current value of the branching from the API
